### PR TITLE
Implemented foreign key support

### PR DESCRIPTION
**Pull request by inbarbarkai**
Implemented foreign key support.

### CODE REVIEW

File: src/database/models.py [hunk 1]
```diff
@@ -18,19 +19,24 @@ class User(Base):
     creation_timestamp = Column(DateTime(timezone=True), nullable=False)
     modify_timestamp = Column(DateTime(timezone=True), nullable=False)
     is_active = Column(Boolean, nullable=False)
-    
+    invitations = relationship("Invitation", back_populates="user")
```
Comment (yarons) [inline src/database/models.py:22]: I'm sorry for the weird question but is it `user` or it should be `user_id`?

**Review by yarons — COMMENTED** [review]
Seems great, I just want your additional comment on what I asked and I'll merge.

File: src/database/models.py (hunk 1 above)
Comment (inbarbarkai) [reply src/database/models.py:22]: `user` is the name of the field to populate with the relevant user in the `Invitation` class.

**Review by yarons — APPROVED** [review]
Checked, it's great.

**Pull request merged**